docs/administration/authentication/active-directory-authentication/index.md

```diff
@@ -44,6 +44,18 @@ Octopus is built on top of HTTP.sys, the same kernel driver that IIS is built on
 When the link is clicked, it redirects to a page which is configured to tell HTTP.sys to issue the browser challenge. The browser and HTTP.sys negotiate the authentication just like an IIS website would. The user principal is then passed to Octopus. Octopus will then query Active Directory for other information about the user.
 :::
 
+### Using Kerberos {#ActiveDirectoryauthentication-UsingKerberos}
+
+By default `IntegratedWindowsAuthentication` will try to use Kerberos first, and then fall back to `NTLM` if it fails.
+
+Please note that this may require additional configuration to work correctly in your network. Generally, this will require a Service Principal Name (SPN) to be set for the `HTTP` service class, using the machine name. For example, if you have an Octopus Deploy hosted on the `od.mydomain.local` domain name, that is installed on a machine called `winserv2019` then the corresponding SPN will be needed:
+
+```powershell
+C:\> setspn.exe -S HTTP/od.mydomain.local winserv2019
+```
+
+For more information about configuration of SPNs [please see this microsoft support article](https://support.microsoft.com/en-us/help/929650/how-to-use-spns-when-you-configure-web-applications-that-are-hosted-on).
```

Comment:
Should we consider using a different url, I found that page quite busy, thoughts?

Reply:
I'd like to get something less busy in. I am not finding a very good substitute though. If there is a better URL that does the following then lets replace this one. It would need to:

- Explain what an SPN is
- How Negotiate works at a high level
- How to troubleshoot difficulties with setting an SPN (remembering that we've given them the basics in the above paragraph)